Heine.Mvc.ActionFilters.Tests/HttpRequestMessages/HttpRequestTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Heine.Mvc.ActionFilters.Extensions;
+using NUnit.Framework;
+
+namespace Heine.Mvc.ActionFilters.Tests.HttpRequestMessages
+{
+    class HttpRequestTests
+    {
+        [Test]
+        public void TestThatStreamIsRemovedFromRequest()
+        {
+            // Arrange
+            var rArray = new byte[50];
```

Comment:
Consider to populate the array with some dummy data. The content will alway be empty now, hence it is hard to tell if the call to `Destruct()` has removed the content or not.

Reply:
yes will fix